microbench/pom.xml

```diff
@@ -52,6 +52,23 @@
       <version>${jmh.version}</version>
       <scope>provided</scope>
     </dependency>
+      <dependency>
+          <groupId>org.alluxio</groupId>
+          <artifactId>alluxio-core-server-master</artifactId>
+          <version>2.9.0-SNAPSHOT</version>
```

Comment:
should this be `${project.version}` and moved up to the first section for internal dependencies? same for the alluxio dep below

also double check identation